lib/inspec/profile_context.rb

```diff
@@ -127,6 +127,7 @@ def load_libraries(libs)
     end
 
     def load_control_file(*args)
+      control_eval_context.skip_file = false
```

Comment:
I'm curious what this is needed for... isn't this false by default? It also feels weird to me for this to be disclosed as a public API on the control eval context, but perhaps I'm missing an idiosyncrasy.

Reply:
Ah, that was a tricky one! From what I could tell (and I could be wrong because this was quite a rabbit hole) is that we only create a `control_eval_context` once and it is set to `false` at that time. Then later on when an `only_if` evaluated to `false` we toggled that value to `true`.

The issue was caused because we never toggled that back to `false`. So I needed a way to set that to `false` between each evaluation/load of a control file. That was the only place I could find that fit the bill.

It also feels weird to me, but I'm not sure how to get around it. Suggestions are welcome!

Reply:
Your explanation is spot-on. The fact that you had to explain it is the problem. :) A one-line code comment above this line explaining why this is necessary will fit the bill.

```
# Set skip_file to false before we read in a new file, just in case the prior file was skipped
```

Reply:
Great! I agree about the comment. It has been added.